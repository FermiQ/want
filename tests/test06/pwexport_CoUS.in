&inputpp
  prefix='cobalt_US'
  outdir='./SCRATCH/'
  pseudo_dir = '../pseudo'
  psfile(1) = 'Co.pbe-nd-rrkjus.UPF'
  single_file = .FALSE.
  ascii = .FALSE.
/
