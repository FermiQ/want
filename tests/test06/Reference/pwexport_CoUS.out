
     Program POST-PROC v.3.0    starts ...
     Today is 30Nov2005 at 20:40:31 

     Reading file cobalt_US.save ...      only dimensions
     read complete

     Reading file cobalt_US.save ...      all except wavefuctions
     read complete

     nbndx  =    10  nbnd   =    10  natomwfc =     6  npwx   =    6495
     nelec  =   9.00  nkb   =    18  ngl    =   10279

     Check: negative/imaginary core charge=   -0.000009    0.000000
